openpype/modules/webserver/webserver_cli.py

```diff
@@ -0,0 +1,151 @@
+import time
+import os
+from datetime import datetime
+import requests
+import json
+
+from openpype.lib import PypeLogger
+
+from .webpublish_routes import (
+    RestApiResource,
+    OpenPypeRestApiResource,
+    WebpublisherBatchPublishEndpoint,
+    WebpublisherTaskPublishEndpoint,
+    WebpublisherHiearchyEndpoint,
+    WebpublisherProjectsEndpoint,
+    BatchStatusEndpoint,
+    PublishesStatusEndpoint
+)
+
+from openpype.api import get_system_settings
+
+SERVER_URL = "http://172.17.0.1:8079"  # machine is not listening on localhost
+
+log = PypeLogger().get_logger("webserver_gui")
+
+
+def run_webserver(*args, **kwargs):
+    """Runs webserver in command line, adds routes."""
+    from openpype.modules import ModulesManager
+
+    manager = ModulesManager()
+    webserver_module = manager.modules_by_name["webserver"]
+    webserver_module.create_server_manager()
+
+    is_webpublish_enabled = False
```

Comment:
Skip whole settings (like mention below to remove it from settings) and btw this should be checked as first thing before anything else...